app/uk/gov/hmrc/mobiletaxcreditssummary/domain/userdata/Claimants.scala

```diff
@@ -18,8 +18,21 @@ package uk.gov.hmrc.mobiletaxcreditssummary.domain.userdata
 
 import play.api.libs.json.{Json, OFormat}
 
-case class Claimants(personalDetails: Person, partnerDetails: Option[Person], children: Seq[Person], ftnaeLink: Option[FtnaeLink] = None)
+case class Claimants(personalDetails: Person, partnerDetails: Option[Person], children: Seq[Person], ftnaeLink: Option[FtnaeLink] = None, reportActualProfit: Option[ReportActualProfit] = None)
 
 object Claimants {
   implicit val format: OFormat[Claimants] = Json.format[Claimants]
 }
+
+case class ReportActualProfit(link: String, endDate: String, userMustReportIncome: Boolean, partnerMustReportIncome: Boolean)
```

Comment:
the end date here any reason this isn't a LocalDateTime and then the writer does the conversion to string for you? Or was this the issue with epoch?

Reply:
As discussed, this field only exists to return a string representation of the date to the app